serving/using-external-dns.md

```diff
@@ -0,0 +1,119 @@
+# Using ExternalDNS to automate DNS setup
+
+[ExternalDNS](https://github.com/kubernetes-incubator/external-dns) is a tool 
+that synchronizes exposed Kubernetes Services and Ingresses with DNS providers.
+
+This doc explains how to set up ExternalDNS within a Knative cluster to 
+automate the process of publishing the domain used in Knative.
+
+## Prerequisite
+
+1. A Kubernetes cluster with [Knative Serving](https://github.com/knative/docs/blob/master/install/README.md) installed.
+1. A public domain that will be used in Knative.
+1. Follow the steps at [Setting up a custom domain](https://github.com/knative/docs/blob/master/serving/using-a-custom-domain.md#edit-using-kubectl) in the "Edit with kubectl" section to make Knative use your domain.
```

Comment:
It might be easier to just pull in the necessary steps to this document instead of linking out to another page, especially since it's only a few steps.

Reply:
Pull them in from the "Setting up a custom domain" document, that is. The "Edit with kubectl" section is pretty short.

Reply:
Done